analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Sonar/SonarExplodedGraphTest.cs

```diff
@@ -917,8 +917,7 @@ public void ExplodedGraph_SwitchStatement_UnsupportedPatternKinds(string pattern
                 }
                 """;
             var context = new ExplodedGraphContext(testInput);
-            var walk = () => context.WalkWithInstructions(2);
-            walk.Should().Throw<Exception>().WithMessage("Expected NumberOfExitBlockReached to be 1, but found 0 (difference of -1).");
+            context.WalkWithExitBlocks(2, 0);
```

Comment:
This started failing for wrong reason. This is the method that should have been used